core/templates/services/assets-backend-api.service.spec.ts

```diff
@@ -363,36 +363,13 @@ describe('Assets Backend API Service', () => {
     }));
   });
 
-  describe('without dev mode settings', () => {
-    beforeAll(() => {
-      spyOnProperty(AssetsBackendApiService, 'GCS_RESOURCE_BUCKET_NAME', 'get')
-        .and.returnValue('');
-      spyOnProperty(AssetsBackendApiService, 'DEV_MODE', 'get')
-        .and.returnValue(false);
-    });
-
-    beforeEach(() => {
-      TestBed.configureTestingModule({
-        imports: [HttpClientTestingModule],
-        providers: [AssetsBackendApiService]
-      });
-    });
-
-    it('should throw an error when is not on dev mode and Google Cloud' +
```

Comment:
Why are we removing this test?

Reply:
Since we now allow GCS in prod mode too it is fine if `GCS_RESOURCE_BUCKET_NAME` is set in prod mode.